package.json

```diff
@@ -11,8 +11,8 @@
   "dependencies": {
     "babel-eslint": "^8.2.2",
     "eslint": "^4.19.1",
-    "eslint-config-airbnb": "^16.1.0",
-    "eslint-config-airbnb-base": "^12.0.0",
+    "eslint-config-airbnb": "17.1.0",
```

Comment:
Can we please keep the `^` here and on the line below to simplify upgrades within the `17.x` version series?